### ISSUE TITLE
CFSSL I need advice

### ISSUE THREAD

**Issue by itkroplis** [cloudflare/cfssl#1069]
I have Goddy wildcard certificate *.firm.com. I want to make with my wildcard certificate for Kubernetes API certificate. That's possible? CFSSL will help do it?

**Comment by darakian** [comment]
Can you refine the question a but more? Do you want to use the cert for `*.firm.com` to create a new cert also for `*.firm.com`? Do you want to make a subordinate CA? What options are set on your certificate?

**Comment by itkroplis** [comment]
Yes. I think so. Make my CA as kubernetes1.firm.com for my K8S cluster 1. But I don't know how it doit.

**Comment by darakian** [comment]
If you're cert has the ability to create subordinate CA's you can use openssl for that with. I made a blog post on this subject recently which might be of some help to you
https://darakian.github.io/2020/01/24/openssl-pki.html

If your cert allows for a sub CA then you can make that and give it the CN `kubernetes1.firm.com` (or whatever) then give that to cfssl which can then make certs for your services.
